test/env/default.sh

```diff
@@ -147,6 +147,7 @@ if [ "$compilername" == "gcc" ]; then
     if [[ "$SUNDIALS_PRECISION" == "double" && "$SUNDIALS_INDEX_SIZE" == "32" ]]; then
         export CFLAGS="${CFLAGS} -Wconversion -Wno-sign-conversion"
         export CXXFLAGS="${CXXFLAGS} -Wconversion -Wno-sign-conversion"
+        export FFLAGS="${FFLAGS} -fbounds-check"
     fi
```

Comment:
Currently is not getting added when the Fortran interfaces are enabled
```suggestion
    fi
    export FFLAGS="${FFLAGS} -fbounds-check"
```